ui/apps/akasha/src/components/pages/profile-feed-page/profile-feed-page.tsx

```diff
@@ -1,31 +1,27 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-import { useParams } from 'react-router-dom';
 import {
   hasOwn,
   mapBeamEntryData,
-  useGetLogin,
+  useGetLoginProfile,
   useRootComponentProps,
 } from '@akashaorg/ui-awf-hooks';
-import { BeamCard, BeamContentResolver, BeamFeed } from '@akashaorg/ui-lib-feed';
-import type { ModalNavigationOptions, Profile } from '@akashaorg/typings/lib/ui';
+import { BeamCard, BeamFeed } from '@akashaorg/ui-lib-feed';
 import ScrollTopWrapper from '@akashaorg/design-system-core/lib/components/ScrollTopWrapper';
 import ScrollTopButton from '@akashaorg/design-system-core/lib/components/ScrollTopButton';
 import Stack from '@akashaorg/design-system-core/lib/components/Stack';
 import { Helmet, HelmetProvider } from 'react-helmet-async';
 
-export type ProfilePageProps = {
-  authenticatedProfile: Profile;
-  showLoginModal: (redirectTo?: { modal: ModalNavigationOptions }) => void;
+type ProfileFeedPageProps = {
+  did: string;
```

Comment:
maybe rename to profileDid, to make it more explicit and we don't confuse it with authenticatedDid